### PR TITLE
Upgrade Sample App to version 2.0.0

### PR DESCRIPTION
**Pull request by aurghob33**
*Issue #, if available:*

*Description of changes:*
Bumped Sample App version.
Added note to CHANGELOG. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,4 +16,4 @@
 
 ### Release 1.0.0-rc.1
 
-* Initial preview release of the QLDB Python Sample Application.
\ No newline at end of file
+* Initial preview release of the QLDB Python Sample Application.
```
Comment (yohanmartin) [inline CHANGELOG.md:19]: Is there a whitespace character change here?

Comment (aurghob33) [reply CHANGELOG.md:19]: No - added no change here: just added at the top.
I tried hard reverting to previous commit and just adding from the top again in vim, same thing showed up in git diff.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+### Release 2.0.0
+* Upgrade Sample App version from 2.00-rc.1 to 2.0.0
```
Comment (yohanmartin) [inline CHANGELOG.md:2]: s/2.00-rc.1/2.0.0-rc.1

Comment (aurghob33) [reply CHANGELOG.md]: Addressed.

**Pull request merged**